apps/studio/src/server/modules/auth/email/__tests__/email.router.test.ts

```diff
@@ -100,6 +101,7 @@ describe("auth.email", () => {
 
     it("should successfully set session on valid OTP", async () => {
```

Comment:
[optional] maybe we want to have a test asserting the wrong OTP -> correct OTP flow, to ensure that our audit logs do capture # of attempts

Reply:
Good idea, added in 6df2f491137cb7321046a905680739bb66f3f381.